frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/transfer-point-popup/transfer-point-popup.component.html

```diff
@@ -185,8 +241,6 @@ <h5 class="popover-header">
                     </div>
```

Comment:
Making the comment here as GitHub doesn't allow comments on all lines: It's probably not new, but I just noticed that the transfer time to the hospital in the transfer point popup is slightly offset to the top. Maybe we can fix this here?

Reply:
Current:
![image](https://user-images.githubusercontent.com/18506183/180623106-f1964c44-d43e-450a-8b64-ba84c198307b.png)
No monospace
![image](https://user-images.githubusercontent.com/18506183/180623108-2b44dbd6-c02f-4e03-8042-b0cba914703d.png)
Both monospace
![image](https://user-images.githubusercontent.com/18506183/180623210-8eb5f84c-ed48-4fef-a9d7-4d50d23b7ddf.png)

The offset is due to the font being monospace. Without monospace different numbers above each other don't look good. -> I let it be like it is now.